Runtime/Core/SelectionPickerSettings.cs

```diff
@@ -93,9 +124,9 @@ public override int GetHashCode()
             {
                 int hashCode = (gameObject != null ? gameObject.GetHashCode() : 0);
                 hashCode = (hashCode * 397) ^ (mesh != null ? mesh.GetHashCode() : 0);
-                hashCode = (hashCode * 397) ^ vertex;
-                hashCode = (hashCode * 397) ^ edge.GetHashCode();
-                hashCode = (hashCode * 397) ^ (face != null ? face.GetHashCode() : 0);
+                hashCode = (hashCode * 397) ^ vertexes.GetHashCode();
+                hashCode = (hashCode * 397) ^ edges.GetHashCode();
```

Comment:
`vertexes` and `edges` both need null checks as well now that they're no longer value types.